### PR TITLE
Fix Nextflow schema file

### PR DESCRIPTION
**Pull request by ewels**
- Remove duplicate parameters
- Remove non-existent parameters
- Remove empty groups
- Reorder, add description, add icons

### CODE REVIEW

**Comment by pditommaso** [comment]
These are all good changes, but I'm sure it will break something! 😄

**Comment by ewels** [comment]
They're hopefully all just things that are broken already 😬

Happy to split into separate PRs if you prefer. The schema is 100% broken and it's surprising that it's ever worked with anything. The other two commits are just things that are depreciated.

File: main.nf [hunk 1]
```diff
@@ -20,21 +20,6 @@
...
-/*
- * Default pipeline parameters. They can be overriden on the command line eg.
- * given `params.foo` specify on the run command line `--foo some_value`.
- */
-
-params.reads = "$baseDir/data/ggal/ggal_gut_{1,2}.fq"
-params.transcriptome = "$baseDir/data/ggal/ggal_1_48850000_49020000.Ggal71.500bpflank.fa"
-params.outdir = "results"
-params.multiqc = "$baseDir/multiqc"
-
```
[the remark this answers is not in the document]
Comment (ewels) [reply main.nf:37]: The only problem I have with that is that it's unclear where the defaults are set. Here or in `nextflow.config`. Less bothered if they are only one or the other.

[the remark this answers is not in the document]
Comment (ewels) [reply main.nf:37]: Note that the schema builder should work fine with them here, even if they're in `nextflow.config` - it just takes the output of `nextflow config -flat` so it's up to Nextflow how to resolve the duplicates.

[the remark this answers is not in the document]
Comment (ewels) [reply main.nf:37]: Ok, and remove from `nextflow.config`?

[the remark this answers is not in the document]
Comment (pditommaso) [reply main.nf:37]: fair enough (not the ones in the profiles tho)

**Comment by ewels** [comment]
Changes included in https://github.com/nextflow-io/rnaseq-nf/pull/19 so can close this in favour of that.

**Pull request closed without merging**